packages/now-next/src/index.ts

```diff
@@ -448,13 +448,30 @@ export const build = async ({
     {}
   );
 
-  const staticDirectoryFiles = onlyStaticDirectory(
-    includeOnlyEntryDirectory(files, entryDirectory),
-    entryDirectory
+  const entryDirectoryFiles = includeOnlyEntryDirectory(files, entryDirectory);
+  const staticDirectoryFiles = filesFromDirectory(
+    entryDirectoryFiles,
+    path.join(entryDirectory, 'static')
+  );
+  const publicDirectoryFiles = filesFromDirectory(
+    entryDirectoryFiles,
+    path.join(entryDirectory, 'public')
+  );
+  const publicFiles = Object.keys(publicDirectoryFiles).reduce(
+    (mappedFiles, file) => ({
+      ...mappedFiles,
+      [file.replace(/public[/\\]+/, '')]: publicDirectoryFiles[file],
```

Comment:
Should this regex be more explicit about what it's doing?

Maybe one of these two:
```suggestion
      [file.replace(/^public[/\\]+/, '')]: publicDirectoryFiles[file],
```
```suggestion
      [file.replace(/.+?public[/\\]+/, '')]: publicDirectoryFiles[file],
```

Reply:
Or is this regex doing neither of these things?

Reply:
@Timer None of the two will properly match this: `www/public/hello.txt`, the output should be `www/hello.txt`